src/System.Data.Common/tests/System/Data/SqlTypes/SqlDecimalTest.cs

```diff
@@ -299,35 +218,27 @@ public void CompareTo()
             Assert.True(_test2.CompareTo(_test3) == 0);
             Assert.True(_test4.CompareTo(SqlDecimal.Null) > 0);
 
-            try
-            {
-                _test1.CompareTo(TestString);
-                Assert.False(true);
-            }
-            catch (ArgumentException e)
-            {
-                Assert.Equal(typeof(ArgumentException), e.GetType());
-            }
+            Assert.Throws<ArgumentException>(() => _test1.CompareTo(TestString));
         }
 
         [Fact]
         public void EqualsMethods()
         {
-            Assert.True(!_test1.Equals(_test2));
-            Assert.True(!_test2.Equals(new SqlString("TEST")));
+            Assert.False(_test1.Equals(_test2));
+            Assert.False(_test2.Equals(new SqlString("TEST")));
             Assert.True(_test2.Equals(_test3));
 
             // Static Equals()-method
             Assert.True(SqlDecimal.Equals(_test2, _test2).Value);
-            Assert.True(!SqlDecimal.Equals(_test1, _test2).Value);
+            Assert.False(SqlDecimal.Equals(_test1, _test2).Value);
 
             // NotEquals
             Assert.True(SqlDecimal.NotEquals(_test1, _test2).Value);
             Assert.True(SqlDecimal.NotEquals(_test4, _test1).Value);
-            Assert.True(!SqlDecimal.NotEquals(_test2, _test3).Value);
+            Assert.False(SqlDecimal.NotEquals(_test2, _test3).Value);
             Assert.True(SqlDecimal.NotEquals(SqlDecimal.Null, _test3).IsNull);
         }
-
+        //TODO
         /* Don't do such environment-dependent test. It will never succeed under Portable.NET and MS.NET
```

Comment:
any suggestions? leave it commented or write up something else? every single `GetHashCode` tests had a `FIXME` saying to use a better testing method,